platform/android/java/lib/src/org/godotengine/godot/GodotView.java

```diff
@@ -94,7 +94,7 @@ public void initInputDevices() {
 	public boolean onTouchEvent(MotionEvent event) {
 		super.onTouchEvent(event);
 		this.detector.onTouchEvent(event);
-		return activity.gotTouchEvent(event);
+		return inputHandler.onTouchEvent(event) || activity.gotTouchEvent(event);
```

Comment:
Should `touch` events be used when the source is a mouse?

Reply:
touch events happands when mouse is clicked even though generic motion event already handled the event, so I use it to my advantage for drag which for some reason does not fire generic motion event